com.unity.render-pipelines.universal/Runtime/2D/Light2DAuthoring.cs

```diff
@@ -26,11 +26,48 @@ void Reset()
         {
             m_ShapePath = new Vector3[] { new Vector3(-0.5f, -0.5f), new Vector3(0.5f, -0.5f), new Vector3(0.5f, 0.5f), new Vector3(-0.5f, 0.5f) };
         }
+        
+        // Rough shape only used in Inspector for quick preview. 
+        static List<Vector2> GetFalloffShape(Vector3[] shapePath)
```

Comment:
I know you only moved this method, but we should take this chance to rename it to what it actually does: GetExtrusionDir().

Reply:
Renamed.